snuba/datasets/configuration/json_schema.py

```diff
@@ -204,6 +204,11 @@ def make_column_schema(
             "type": "string",
             "description": "The distributed table name in distributed ClickHouse",
         },
+        "part_format": {
```

Comment:
This is actually partition format, not part format. I know the code is wrong today in referring to these as part. But it should be partitions. Since you are creating new dataset configuration, may I suggest we fix the name now?
```suggestion
        "partition_format": {
```